### DISCUSSION TITLE
r/china-travel-layoff-trip-day12-xiuning-mulihong

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#81] [Announcements]
# r/china-travel-layoff-trip-day12-xiuning-mulihong

这里是皖南最美天空村落，云端之城

https://blog.ops-coffee.cn/r/china-travel-layoff-trip-day12-xiuning-mulihong

**Comment by shuihunX** [comment]
豫西山区...
已走出豫，未来会更好。
